corehq/apps/zapier/tests.py

```diff
@@ -0,0 +1,187 @@
+from django.test.testcases import TestCase
+from mock import patch
+from tastypie.models import ApiKey
+
+from corehq.apps.app_manager.models import Application, Module
+from corehq.apps.domain.models import Domain
+from corehq.apps.receiverwrapper.util import submit_form_locally
+from corehq.apps.users.models import WebUser
+from corehq.apps.zapier import consts
+from corehq.apps.zapier.models import Subscription
+from corehq.toggles import ZAPIER_INTEGRATION
+
+XFORM = """
+    <h:html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:orx="http://openrosa.org/jr/xforms"
+    xmlns="http://www.w3.org/2002/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+    xmlns:jr="http://openrosa.org/javarosa" xmlns:vellum="http://commcarehq.org/xforms/vellum">
+        <h:head>
+            <h:title>Checkup</h:title>
+            <model>
+                <instance>
+                    <data xmlns:jrm="http://dev.commcarehq.org/jr/xforms"
+                    xmlns="https://www.commcarehq.org/test/zapier/"
+                    uiVersion="1" version="1" name="Checkup">
+                        <question1/>
+                        <question2/>
+                        <stub/>
+                    </data>
+                </instance>
+                <bind nodeset="/data/question1" type="xsd:string"/>
+                <bind nodeset="/data/question2" type="xsd:string"/>
+                <itext>
+                    <translation lang="en" default="">
+                        <text id="question1-label">
+                            <value>question1</value>
+                        </text>
+                        <text id="question2-label">
+                            <value>question2</value>
+                        </text>
+                    </translation>
+                </itext>
+            </model>
+        </h:head>
+        <h:body>
+            <input ref="/data/question1">
+                <label ref="jr:itext('question1-label')"/>
+            </input>
+            <input ref="/data/question2">
+                <label ref="jr:itext('question2-label')"/>
+            </input>
+        </h:body>
+    </h:html>
+"""
+
+FORM_XMLNS = "https://www.commcarehq.org/test/zapier/"
+XFORM_XML_TEMPLATE = """<?xml version='1.0' ?>
+<data xmlns:jrm="http://dev.commcarehq.org/jr/xforms" xmlns="%s">
+    <woman_name>Alpha</woman_name>
+    <husband_name>Beta</husband_name>
+    <meta>
+        <deviceID>O2XLT0WZW97W1A91E2W1Y0NJG</deviceID>
+        <timeStart>2011-10-01T15:25:18.404-04</timeStart>
+        <timeEnd>2011-10-01T15:26:29.551-04</timeEnd>
+        <username>admin</username>
+        <userID>{}</userID>
+        <instanceID>{}</instanceID>
+    </meta>
+</data>
+""" % FORM_XMLNS
+ZAPIER_URL = "https://zapier.com/hooks/standard/1387607/5ccf35a5a1944fc9bfdd2c94c28c9885/"
+TEST_DOMAIN = 'test-domain'
+MockResponse = namedtuple('MockResponse', 'status_code reason')
+
+
+class TestZapierIntegration(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
I think django 1.8 and above require you to call the superclass methods for `setUp/tearDown(Class)`. can you add those?